.github/workflows/ci.yaml

```diff
@@ -104,6 +104,17 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: ${{ needs.build.outputs.artifact-name }}
+      - name: Free disk space
```

Comment:
interesting! why is this need?

Reply:
There was issue on CI: some tests were not able to start because of "No space left on device" error. You can c[heck here ](https://github.com/canonical/mongodb-operator/actions/runs/6498579157) for example. This change frees some space

<img width="784" alt="Screenshot 2023-10-13 at 11 05 02" src="https://github.com/canonical/mongodb-operator/assets/132273757/58a9efac-e8e9-4d2e-acd8-cdf713283333">